ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -35,6 +35,8 @@ QObject {
     readonly property alias dappsModel: dappsProvider.dappsModel
     readonly property alias requestHandler: requestHandler
 
+    readonly property bool isServiceAvailableForAddressSelection: dappsProvider.supportedAccountsModel.count
```

Comment:
We're assuming here the `dappsProvider.supportedAccountsModel` is and will always be a `ListModel`. This can be changed to `dappsProvider.supportedAccountsModel.ModelCount.count`

Reply:
To be fixed